specification/sections/02-providers.md

```diff
@@ -250,6 +253,15 @@ class MyProvider implements Provider, AutoDisposable {
   }
 ```
 
+#### Requirement 2.5.2
+
+> After a provider's shutdown function has terminated successfully, the provider's state **MUST** revert to its uninitialized state.
+
+If a provider requires initialization, once it's shut down, it should transition to its initial `NOT_READY` state, so it can be reinitialized.
```

Comment:
```suggestion
If a provider requires initialization, once it's shut down, it must transition to its initial `NOT_READY` state, so it can be reinitialized.
```